aten/src/ATen/native/mkldnn/xpu/detail/oneDNNContext.cpp

```diff
@@ -0,0 +1,26 @@
+#include <ATen/native/mkldnn/xpu/detail/oneDNNContext.h>
+#include <ATen/native/mkldnn/xpu/detail/Utils.h>
+
+/* XXX WARNING:
+ * Do NOT put any kernels or call any device binaries here!
+ * Only maintain oneDNN runtime states in this file.
+ * */
+namespace at::native{
+namespace xpu {
+namespace onednn {
```

Comment:
namespace `at::native::xpu::onednn` is so long. If the user uses `using namespace at::native;` in his code,  it is easy to have a namespace conflict with `c10::xpu` and `at::xpu` when call `xpu::onednn::func`. Because the compiler doesn't know which `xpu` should be referred to(`c10::xpu`, `at::xpu` or `at::native::xpu`)? Could you simplify this namespace to `at::native::onednn`. The user can call these funcs via `onednn::func` instead of `func` under `using namespace at::native::xpu::onednn;`

Reply:
Thanks for suggestion, the namespace has been changed to `at::native::onednn`